src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -130,8 +157,160 @@ object Config {
   def sourceHasHeader(s: Source): Boolean = header.findFirstIn(s.content).nonEmpty
 }
 
+case class BaseConfig(moduleName: String = ConfigDefaults.DefaultModuleName,
+                      includeDirs: Vector[Path] = ConfigDefaults.DefaultIncludeDirs.map(_.toPath).toVector,
```

Comment:
This file seems to have a lot of repetition, but I don't see immediately how to avoid it

Reply:
do you mean due to the similarity of `Config` and `BaseConfig` and so on or also in terms of code that instances them? In the latter case, I would argue that this might even be desirable for code readability. We explicitly separate the handling of the 3 modes to just achieve that. I agree that it's similar but still there are differences on how exactly we e.g. create the package info map. I quite like to read up on this on a per-mode basis

Reply:
Yes, I meant the sharing of fields between `BaseConfig` and `Config`